frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -81,3 +81,15 @@ export const calcPVsCapacity = (pvs: K8sResourceKind[]): number =>
 
 export const getSCAvailablePVs = (pvsData: K8sResourceKind[], sc: string): K8sResourceKind[] =>
   pvsData.filter((pv) => getPVStorageClass(pv) === sc && pv.status.phase === status.AVAILABLE);
+
+export const getSCNamesUsingPool = (
+  scData: ListKind<K8sResourceKind>,
+  poolName: string,
+): string[] =>
+  scData.items?.reduce((scNames, sc) => {
+    if (_.get(sc, 'parameters.pool') === poolName) scNames.push(sc.metadata?.name);
+    return scNames;
+  }, []);
```

Comment:
Do not put this here, if its not going to be used by components other than pool.
```suggestion
export const getSCNamesUsingPool = (
  scData: ListKind<K8sResourceKind>,
  poolName: string,
): string[] =>
  scData.items?.reduce((scNames, sc) => {
    if (sc?.parameters?.pool === poolName) scNames.push(sc.metadata?.name);
    return scNames;
  }, []);
```

Reply:
Ah! got the point